X86/X86MachineInstructionRaiser.cpp

```diff
@@ -3000,6 +3037,10 @@ bool X86MachineInstructionRaiser::raiseDirectBranchMachineInstr(
   return true;
 }
 
+bool X86MachineInstructionRaiser::hasInstrPrefix(const MachineInstr &MI, StringRef name) const {
```

Comment:
The term 'prefix' in the context of Intel-64 instructions conjures up instruction prefixes such as `REP`. So, I'd rename this function as `instrOpCodeStringStartsWith` to be unambiguous.

Reply:
Yeah also came to my mind.